subcommittees/mentoring/PracticumProposal.md

```diff
@@ -0,0 +1,196 @@
+
+Second, we are expecting that the people we admit to be our teachers already know certain things.
+This might not be the case - we could have people who are great at Unix but who know little about
+Git, and people with R knowledge who would not be able to manoeuvre on the command line. Thus, there
+might be gaps in their knowledge, which the trainees themselves may or may not be aware of. By having
+them work through the core lesson material, we ensure that the trainees have at least a minimum level
+of knowledge in all of our core subjects. 
+
+Third, many instructors don’t really know too many others within the Software and Data Carpentry
+organizations. By having people working in teams during training, we strengthen between-instructor
+connections which may lead to more discussions between instructors at a later stage. Also, by having
+people already within the community mentoring future instructors, they get to know people already
+inside of the organization. 
+
+Fourth, as a bonus we get a lot of people working through the material, which can only be a benefit to
+the lesson development process. It might draw more people into helping out with lesson development, and
+could lower the bar for people developing new lessons.
+
+With these goals in mind, we have developed a post-instructor training practicum.  
+
+## Practicum elements
+
+The instructor training practicum aims to facilitate the transition from the introductory instructor
+training  to actually teaching workshops, by doing the following activities: 
+
+### Mentoring
+
+Future instructors, or trainees, form small groups and are assigned a mentor.  Over a period of a 3-4
+weeks, they work through the core material for Software and Data Carpentry on their own, as if they
+were workshop participants.  Each week, the group meets to discuss their experiences with the lesson
+material and the mentor receives feedback and provides guidance.  The primary goal is not that the
+trainees should be prepared enough to teach the module, but to become familiar with it and gain skills.  
+
+### Mini-lessons
+
+Each trainee chooses a topic relevant to Software or Data Carpentry (can be one of our lessons or
+their own provided their mentor approves) and teaches a 1 hour lesson at their local institution to an
+audience of their choice.  This lesson should be at least partially observed by a mentor and feedback
+provided.  Whenever possible, the mentor observing the mini-lesson should be the mentor of the previous
+mentoring element.  
+
+### General comments
+
+These two elements need not happen consecutively, but in the first implementation of this practicum
+they will be consecutive, not parallel (see timeline).  
+
+These two elements will be required to become a fully badged Software or Data Carpentry Instructor.  
+
+## Implementation Logistics
+
+### Mentoring Groups
+
+For the first practicum, all recent participants in instructor training (online 8 week, December 2 day)
```

Comment:
Suggest to rephrase to be specific and avoid confusion: "all participants of the two Pilot Online Instructor Training sessions"